src/agent/MessageSender.ts

```diff
@@ -26,16 +35,22 @@ export class MessageSender {
   }
 
   public async packMessage(outboundMessage: OutboundMessage): Promise<OutboundPackage> {
-    return this.envelopeService.packMessage(outboundMessage)
+    const { connection, payload } = outboundMessage
+    const { verkey, theirKey } = connection
+    const endpoint = this.transportService.findEndpoint(connection)
+    const message = payload.toJSON()
+    this.logger.info('outboundMessage', { verkey, theirKey, message })
```

Comment:
Good point. I agree. I don't like it either. 

For now, I change this one to `debug` because it logs an unencrypted message. It seems outbound encrypted messages are logged in outbound transporters now. That we could change to `trace`. 

This logic will likely change, but I'll think about that more during the implementation. We can revise that in the next PR.